sdk/formrecognizer/ai-form-recognizer/samples/javascript/recognizeBusinessCard.js

```diff
@@ -73,6 +73,10 @@ async function main() {
  * processing.
  */
 function printSimpleArrayField(businessCard, fieldName) {
+  if (businessCard.fields[fieldName] == null) {
```

Comment:
```suggestion
  if (!businessCard.fields[fieldName]) {
```

Seems a little nicer in this case.